### PR TITLE
Shotgun tube toggle hotkey

### PR DESCRIPTION
**Pull request by Blundir**
# About the pull request

Adds hotkey to toggling multi magazine on multi tube shotgun.

# Explain why it's good for the game

Having hotkeys for combat related actions is very important.

</details>

# Changelog
:cl:
add: added shotgun tube toggle hotkey
/:cl:

### CODE REVIEW

File: code/datums/keybinding/human_combat.dm [hunk 1]
```diff
@@ -190,3 +190,19 @@
...
+	name = "toggle_shotgun_tube"
+	full_name = "Toggle Shotgun Tube"
+	keybind_signal = COMSIG_KB_HUMAN_WEAPON_SHOTGUN_TUBE
+
+/datum/keybinding/human/combat/toggle_shotgun_tube/down(client/user)
+	. = ..()
+	if(.)
+		return
+	var/mob/living/carbon/human/human = user.mob
+	var/obj/item/weapon/gun/shotgun/pump/dual_tube/held_item = human.get_held_item()
```
Comment (Zonespace27) [inline code/datums/keybinding/human_combat.dm:206]: You need to check if what they're holding exists and is this type

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/datums/keybinding/human_combat.dm (hunk 1 above)
Comment (fira) [reply code/datums/keybinding/human_combat.dm:206]: also if client is valid

Comment (Blundir) [reply code/datums/keybinding/human_combat.dm:206]: also why all this isn't a case for other hotkeys?

File: code/datums/keybinding/human_combat.dm [hunk 2]
```diff
@@ -190,3 +190,20 @@
...
+	if(istype(held_item, /obj/item/weapon/gun/shotgun/pump/dual_tube))
```
Comment (Zonespace27) [inline code/datums/keybinding/human_combat.dm:207]:
```suggestion
	if(istype(held_item))
```

**Pull request merged**